### PR TITLE
Removes the augment painter, moves the function to spraycan rightclicking.

### PR DESCRIPTION
**Pull request by zxaber**
## About The Pull Request
Right-clicking a robotic body part with a spraycan will open a radial menu and allow for coloring the body part just like the augment painter did. This costs five charge off the spraycan, meaning a single can will paint six augments.

The augment painter has been removed from the game files and from all maps.

## Why It's Good For The Game
I can count on one hand the number of times I've *seen* anyone bother painting augments, and I play robotics more than any other role. Generally, the machine just sits in the way until I need the space and I shove it into the halls to get lost somewhere.

For anyone that does enjoy their colored robot parts, a spray can is obtainable from an autolathe.

## Changelog

:cl:
del: The augment painter has been removed from the game.
add: Spraycans can now change the visual style of robotic augments via right-click.
/:cl:

This time with an updated branch.

### CODE REVIEW

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -778,6 +778,31 @@
...
+			for(var/skin_option in style_list_icons)
+				var/image/part_image = image(icon = style_list_icons[skin_option], icon_state = limb.icon_state)
+				skins += list("[skin_option]" = part_image)
+			var/choice = show_radial_menu(user, src, skins, require_near = TRUE)
+			if(choice && (use_charges(user, 5, requires_full = FALSE) == 5))
+				playsound(user.loc, 'sound/effects/spray.ogg', 5, TRUE, 5)
+				limb.icon = style_list_icons[choice]
+			return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+
+	. = ..()
```
Comment (Mothblocks) [inline code/game/objects/items/crayons.dm:804]: Never return `..()` from secondary attacks, its behavior I'm going to remove someday.

```suggestion
	return SECONDARY_ATTACK_CONTINUE_ATTACK_CHAIN
```

File: code/game/objects/items/crayons.dm [hunk 2]
```diff
@@ -778,6 +778,31 @@
 
 	. = ..()
 
+/obj/item/toy/crayon/spraycan/afterattack_secondary(atom/target, mob/user, proximity, params)
+	if(!proximity)
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+	if(is_capped)
+		to_chat(user, span_warning("Take the cap off first!"))
```
Comment (Mothblocks) [inline code/game/objects/items/crayons.dm:785]:
```suggestion
		balloon_alert(user, "take the cap off first!"))
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/game/objects/items/crayons.dm [hunk 3]
```diff
@@ -778,6 +778,31 @@
...
+			for(var/skin_option in style_list_icons)
+				var/image/part_image = image(icon = style_list_icons[skin_option], icon_state = limb.icon_state)
+				skins += list("[skin_option]" = part_image)
+			var/choice = show_radial_menu(user, src, skins, require_near = TRUE)
+			if(choice && (use_charges(user, 5, requires_full = FALSE) == 5))
+				playsound(user.loc, 'sound/effects/spray.ogg', 5, TRUE, 5)
+				limb.icon = style_list_icons[choice]
+			return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+
+	return SECONDARY_ATTACK_CONTINUE_ATTACK_CHAIN
```
Comment (Mothblocks) [inline code/game/objects/items/crayons.dm:804]:
```suggestion
	return SECONDARY_ATTACK_CONTINUE_CHAIN
```

my bad

**Comment by AndroidMK2** [comment]
Would be nice to be able to change the augment color too rather than just it's style.

**Review by Maurukas — APPROVED** [review]

**Pull request merged**